do-batch

```diff
@@ -130,7 +153,7 @@ for charm in $charms; do
       git commit --amend -F $commit_msg_file
       git_review
     elif [[ "$all_params" == *--do-review* ]]; then
-      git_review
+      git_review || true
```

Comment:
Is it a workaround in case there is no change and thus there is no review to be created?

Reply:
Yes, it just allows it to move on to the next charm; otherwise it fails out which is not too useful.